app/scripts/services/tile-proxy.js

```diff
@@ -609,15 +609,19 @@ function fetchEither(url, callback, textOrJson, pubSub) {
   requestsInFlight += 1;
   pubSub.publish('requestSent', url);
 
-  let mime;
+  let mime = null;
   if (textOrJson === 'text') {
-    mime = 'text/plain';
+    mime = null;
```

Comment:
Seems unnecessary because you initialize `mime` with `null`.

Just curious, why is the `'text/plain'` mime type harmful?

Reply:
Nevermind, I just read your PR description, which answers my question :)

I think the entire if statement can be removed